PubnativeLite/PubnativeLite/Location/PNLiteLocationManager.m

```diff
@@ -48,6 +48,18 @@ - (id)init {
     return self;
 }
 
+static BOOL locationUpdatesEnabled;
++ (BOOL) locationUpdatesEnabled {
+    @synchronized(self) { return locationUpdatesEnabled; }
+}
+
++ (void) setLocationUpdatesEnabled:(BOOL)enabled {
```

Comment:
Not an issue but for the sake of code convention let's remove the space between ) and method name 😂

Reply:
done :))